torntools/scripts/content/education/ttEducation.js

```diff
@@ -0,0 +1,12 @@
+requireDatabase().then(() => {
+	console.log("TT - Education");
+	hideCompletedCategories();
+});
+
+function hideCompletedCategories() {
+	if (window.location.href.indexOf("step=main") > -1) {
+		for (let category of doc.find(".education").findAll(".ajax-act")) {
```

Comment:
The `find` + `findAll` should be combined into 1 `findAll`.

Reply:
If the user is currently doing a course, there will be another element of class ajax-act outside class education element. So, find(."education") is required. Rather than wrapping it around in another if, I found it better.

Reply:
You can do `.education .ajax-act` though.

Reply:
Done